### ISSUE TITLE
Some textures won't show up...

### ISSUE THREAD

**Issue by User3047** [MightyPork/rpw#40]
I imported already made pack and it didn't show all the textures for the blocks. Is the problem in this specific pack or it's something else?
![github_minecraft_issue](https://cloud.githubusercontent.com/assets/12634205/7849641/d570329c-04d7-11e5-8a55-c5d2b759692e.png)

EDIT: This is only one example, I wanted to change this pack and these doors were the first thing that I noticed, but there are more than a few blocks that didn't load/show up.

**Comment by Jiggy-12321** [comment]
You may have imported it wrong. I'll let MightyPork look at it, but it works fine on my end.

**Comment by MightyPork** [comment]
I'll need the pack you imported to try it. Right now, my guess is that the pack is for older Minecraft version and the door textures got renamed, but that's just a theory. RPW's runtime log from when you imported the pack would tell much more.

**Comment by MightyPork** [comment]
Another thing to try, look at the project using your file manager, and try to locate the doors there. They could end up in `extra_files` if my previous theory was correct. Then you'd have to rename them to the right names and move to `project_files` (you can still use the file manager, easier than importing one-by-one in RPW)

**Comment by User3047** [comment]
I sent you to your mail (ondra@ondrovo.com) the pack and the log when loading this certain pack. Since I had issues with Gmail security I had to un-zip the pack, so you have it now in folder, but it was in .zip originally. I did find the door textures, but they don't look like the in-game model. The upper half is full, and in-game it is nearly 90% empty. Hope this (and mail) helps solving my problem.
